map/src/infoblock/components/wpt/WptDetails.jsx

```diff
@@ -0,0 +1,350 @@
+            <Box className={styles.wptCategory}>
+                <ListItemIcon sx={{ minWidth: 'auto', fill: color }}>
+                    <DirectionIcon />
+                </ListItemIcon>
+                <ListItemText>
+                    <Typography sx={{ color: color + '!important' }} className={styles.wptCategoryText} noWrap>
+                        {locDist(wpt, location) ?? 'No distance'}
+                    </Typography>
+                </ListItemText>
+            </Box>
+        );
+    };
+
+    return (
+        <>
+            <Box
+                minWidth={ctx.infoBlockWidth}
+                maxWidth={ctx.infoBlockWidth}
+                sx={{ height: 'auto', overflowX: 'hidden' }}
+            >
+                <Header />
+                {wpt !== null && (
+                    <ListItemText id={getId()}>
+                        <Box className={styles.topContainer}>
+                            <MenuItemsTitle maxLines={3} className={styles.name}>
+                                <Typography className={styles.name}>
+                                    {wpt.type?.isPoi ? (
+                                        <Link href={wpt.osmUrl} target="_blank" underline="none">
+                                            {wpt.name ? wpt.poiType + ': ' + wpt.name : wpt.poiType}
+                                        </Link>
+                                    ) : (
+                                        wpt.name ?? 'No name'
+                                    )}
+                                </Typography>
+                            </MenuItemsTitle>
+                            {wpt.icon && <WptIcon />}
+                        </Box>
+                        {wpt?.category && <WptCategory />}
+                        {wpt.latlon && currentLoc && <WptLoc wpt={wpt} location={currentLoc} />}
+                        {wpt?.address && <WptAddress />}
+                        <WptDetailsButtons wpt={wpt} isDetails={isDetails} />
+                        {wpt.desc && (
+                            <WptTagInfo
+                                key={'desc'}
+                                baseTag={{
+                                    icon: <DescriptionIcon />,
+                                    name: t('shared_string_description'),
+                                    value: wpt.desc,
+                                    isDesc: true,
+                                }}
+                            />
+                        )}
+                        {wpt.time && (
+                            <WptTagInfo
+                                key={'time'}
+                                baseTag={{
+                                    icon: <TimeIcon />,
+                                    name: t('date_of_creation'),
+                                    value: formatTime(wpt.time),
+                                }}
+                            />
+                        )}
+                        {wpt.hidden && (
```

Comment:
Please clarify what is `hidden` and why "Hidden: yes" appears on all favorites regardless they were uploaded from gpx or were created in OsmAnd, and even they are not hidden on the map currently.